skyrl-train/tests/cpu/generators/test_skyrl_gym_generator_chat_templating.py

```diff
@@ -72,6 +71,11 @@ def mock_generate(input_batch):
 
     mock_llm.generate = AsyncMock(side_effect=mock_generate)
     # Create a mock generator config
+
+    chat_template_config = None
+    if "Qwen3" in model_name:
+        chat_template_config = {"source": "name", "name_or_path": "qwen3_without_thinking"}
```

Comment:
```suggestion
    if "Qwen3" in model_name:
        chat_template_config = {"source": "name", "name_or_path": "qwen3_without_thinking"}
    else:
        chat_template_config = {"source": "name", "name_or_path": None}
```

Would this be a more realistic config? Since for other models we would still have the config, but just the default values (i.e. `None`)